### ISSUE TITLE
Mysql access

### ISSUE THREAD

**Issue by iantearle** [reddingwebpro/cognacbox#6]
Just got this up and running, but facing problems accessing the database, assumed it would be the same as scotch box "root" and "root" but not working, is it set as default or do I need to create a new user?

**Comment by reddingwebpro** [comment]
root and root is the correct username if accessing from within the box or an application running on the box. If you need to access it from your workstation then you'll need to SSH in using these instructions: https://www.reddingitpro.com/2021/01/08/connect-mysql-workbench-to-vagrant-box-over-ssl/

**Comment by iantearle** [comment]
Thanks for confirming, I ran into a "ECDSA host key" which is where I was hitting, I'm moving from scotch box to this. Got the first key sorted, then struggling with ssh, keep hitting vagrant@127.0.0.1: Permission denied (publickey). 

Ran through your instructions, thank you. Key component was step 1. Utilising that key, interesting how scotch box allowed with password? I guess that's security change for you?!

Thanks for your help!
